site/src/common/blocks/MediaGalleryBlock.tsx

```diff
@@ -0,0 +1,139 @@
+import "swiper/css";
+import "swiper/css/navigation";
+
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { MediaGalleryBlockData } from "@src/blocks.generated";
+import { MediaBlock } from "@src/common/blocks/MediaBlock";
+import { Typography } from "@src/common/components/Typography";
+import { PageLayout } from "@src/layout/PageLayout";
+import styled from "styled-components";
+import { Navigation } from "swiper/modules";
+import { Swiper, SwiperSlide } from "swiper/react";
+
+type MediaGalleryBlockProps = PropsWithData<MediaGalleryBlockData>;
+
+export const MediaGalleryBlock = withPreview(
+    ({ data }: MediaGalleryBlockProps) => {
+        const aspectRatioValues = data.aspectRatio.split("x");
+
+        return (
+            <SwiperWrapper
+                modules={[Navigation]}
+                slidesPerView={1}
+                slidesPerGroup={1}
+                navigation
+                longSwipesRatio={0.1}
+                threshold={3}
+                allowTouchMove
+                watchOverflow
+                $aspectRatioHorizontal={aspectRatioValues[0]}
+                $aspectRatioVertical={aspectRatioValues[1]}
+            >
+                {data.items.blocks.map((block) => (
+                    <SwiperSlide key={block.key}>
+                        <MediaBlock data={block.props.media} aspectRatio={data.aspectRatio} />
+                        <MediaCaption variant="p200" component="p">
```

Comment:
```suggestion
                        <MediaCaption variant="p200">
```

The `component` is set automatically to `p` when you set this variant.

Reply:
done